python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -467,6 +480,15 @@ def toTable(self):
                 expIdMaskList.append(self.expIdMask[amp])
                 rawMeanList.append(self.rawMeans[amp])
                 rawVarianceList.append(self.rawVariances[amp])
+
+                correlationShape = np.array(self.rawXcorrs[amp]).shape
+                if nObs != correlationShape[0]:
+                    if correlationShape[0] == np.sum(self.expIdMask[amp]):
+                        # Repack data.
+                        self.repackCorrelations(amp, correlationShape)
+                    else:
+                        raise ValueError("Could not coerce rawXcorrs into appropriate shape.")
```

Comment:
Do you need to expand the error message here too?

Reply:
Ah, good catch.  Thanks!